Weapon/src/main/java/dk/sdu/se/f23/InVasion/weapon/WeaponPlugin.java

```diff
@@ -1,22 +1,39 @@
 package dk.sdu.se.f23.InVasion.weapon;
 
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
 import dk.sdu.se.f23.InVasion.common.data.Entity;
 import dk.sdu.se.f23.InVasion.common.data.GameData;
 import dk.sdu.se.f23.InVasion.common.data.ProcessAt;
 import dk.sdu.se.f23.InVasion.common.data.World;
 import dk.sdu.se.f23.InVasion.common.services.EntityProcessingService;
 import dk.sdu.se.f23.InVasion.common.services.PluginService;
 
+import java.util.ArrayList;
+import java.util.Objects;
+
 public class WeaponPlugin implements PluginService {
     private String weaponName = "Steven";
+    private Texture texture = new Texture(Gdx.files.internal("Weapon/src/main/resources/TOWER.png"));
```

Comment:
```suggestion
    private Texture texture = new Texture(Gdx.files.internal("resources/TOWER.png"));
```

Reply:
See i agree that it looks better. But it does not work for some reason. Enemy and player have also been changed to use the repo root, maven do not like it